doc/modules/linear_model.rst

```diff
@@ -897,8 +897,15 @@ following cost function:
 .. math::
     :name: regularized-logistic-loss
 
-    \min_{w} C \sum_{i=1}^n \left(-y_i \log(\hat{p}(X_i)) - (1 - y_i) \log(1 - \hat{p}(X_i))\right) + r(w).
+    \min_{w} C \sum_{i=1}^n s_i \left(-y_i \log(\hat{p}(X_i)) - (1 - y_i) \log(1 - \hat{p}(X_i))\right) + r(w).
 
+where :math:`{s_i}` corresponds to the weights assigned by the user to a
+specific training sample (the vector :math:`s` is formed by element-wise
+multiplication of the class weights and sample weights). :math:`{y_i}` is the
```

Comment:
From `:math:`{y_i}` it‘s a repetition what has already been said above or will be said below (the penalty). Please remove.

Reply:
Oh, okay, only now I understand your intention from your previous suggestion. I will remove.